lib/osf-components/addon/components/cedar-metadata-editor/template.hbs

```diff
@@ -0,0 +1,13 @@
+<div>
+    <cedar-embeddable-editor
+        config={{this.cedarConfig}}
+        templateObject={{this.templateObject}}
+        instanceObject={{this.instanceObject}}
```

Comment:
Noticed that the argument values don't match what's in the component file. Are we passing in the `templateObj` and `instanceObj` that's defined in the `Args` interface?